WORKSPACE

```diff
@@ -61,10 +60,12 @@ go_rules_dependencies()
 
 go_register_toolchains()
 
-go_image_repositories()
+load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies")
```

Comment:
Why the ordering change?

Reply:
To match the recommended way to load gazelle - I thought it may be important. https://github.com/bazelbuild/bazel-gazelle#running-gazelle-with-bazel